src/redux/books/books.js

```diff
@@ -0,0 +1,26 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
```

Comment:
- [ ] **Required:** Your work is outstanding and works correctly. However, you are required to use the `ducks pattern` to define your action types. For example, your add action type could be re-written as `const ADD_BOOK = "bookstore/books/ADD_BOOK"`.
Kindly do this for `REMOVE_BOOK` as well and for every action type, you create going forward. This is to distinguish cases where different parts of your codebase have a similar action type. For example, if your `redux/books...` and `redux/categories...` all had an `ADD_BOOK` action type. I hope it makes sense. 😄